bluesky/plans.py

```diff
@@ -2565,7 +2565,8 @@ def adaptive_scan(detectors, target_field, motor, start, stop,
     --------
     :func:`bluesky.plans.relative_adaptive_scan`
     """
-    assert 0 < min_step < max_step, "min_step and max_step must meet condition of max_step > min_step > 0"
+    if not 0 < min_step < max_step:
+        raise ValueError("min_step and max_step must meet condition of max_step > min_step > 0")
```

Comment:
This needs to be broken into two lines to fit inside the PEP8 79-character limit. This is one nice way to do this:

```python
        raise ValueError("min_step and max_step must meet condition of "
                         "max_step > min_step > 0")
```

Reply:
good catch.